client/src/components/AutoSleepToggle/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react'
+import NoSleep from 'nosleep.js'
+import { isMobile } from '@/utils'
+import { Toggle } from "@/components";
+import './style.scss'
+
+const AutoSleepToggle = ({ noSleepEnabled, setNoSleepEnabled }: { noSleepEnabled: boolean, setNoSleepEnabled: (arg0: boolean) => void }): JSX.Element => {
```

Comment:
I think we should change these props (`noSleepEnabled, setNoSleepEnabled`) to something more intuitive.